spec/index.bs

```diff
@@ -5976,6 +5976,28 @@ A {{GPUBindGroup}} object has the following internal slots:
         associated with lists of the [=internal usage=] flags.
 </dl>
 
+<div algorithm>
+    The <dfn for=GPUBindGroup>bound buffer ranges</dfn> of a {{GPUBindGroup}} |bindGroup|,
+    given [=list=]&lt;GPUBufferDynamicOffset&gt; |dynamicOffsets|, are computed as follows:
+
+    1. Let |result| be a new [=set=]&lt;({{GPUBindGroupLayoutEntry}}, {{GPUBufferBinding}})&gt;.
+    1. Let |dynamicOffsetIndex| be 0.
+    1. For each {{GPUBindGroupEntry}} |bindGroupEntry| in |bindGroup|.{{GPUBindGroup/[[entries]]}},
+        sorted by |bindGroupEntry|.{{GPUBindGroupEntry/binding}}:
+        1. Let |bindGroupLayoutEntry| be
+            |bindGroup|.{{GPUBindGroup/[[layout]]}}.{{GPUBindGroupLayout/[[entryMap]]}}[|bindGroupEntry|.{{GPUBindGroupEntry/binding}}].
+        1. If |bindGroupLayoutEntry|.{{GPUBindGroupLayoutEntry/buffer}} is not
+            [=map/exists|provided=], **continue**.
+        1. Let |bound| be a copy of |bindGroupEntry|.{{GPUBindGroupEntry/resource}}.
```

Comment:
I'm not gonna even call this a nit, just point out that the "copy" is only necessary when the buffer has a dynamic offset. I expect that any reasonable implementation isn't going to blindly be following this algorithm in their implementation, though, so it really doesn't matter.

Reply:
It's only be a copy of the buffer pointer, offset, and size, so 24 bytes. Something that in C++ you'd probably be holding by value and copying around anyway, we just have to write this because in the spec the object is a sort of implicitly a reference (somewhat unfortunately).